x-pack/solutions/security/plugins/security_solution/public/exceptions/components/list_exception_items/index.tsx

```diff
@@ -74,13 +71,19 @@ const ListExceptionItemsComponent: FC<ListExceptionItemsProps> = ({
       : i18n.EXCEPTION_ITEM_CARD_DELETE_LABEL;
   }, [listType]);
 
+  const exceptionsTitle = useMemo(() => {
+    return listType === ExceptionListTypeEnum.ENDPOINT
+      ? i18n.EXCEPTION_UTILITY_ENDPOINT_TITLE
+      : i18n.EXCEPTION_UTILITY_TITLE;
+  }, [listType]);
+
```

Comment:
nit: It doesn't look like it's necessary to memoize the title nor it brings performance benefits.

Reply:
agree, done, also for the other consts
ab8e90a8902c2fe8afb50ea65dba0c4eee2168ad